pkg/kubectl/cmd/log.go

```diff
@@ -85,6 +85,7 @@ func NewCmdLog(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	cmd.Flags().Bool("interactive", true, "If true, prompt the user for input when required. Default true.")
 	cmd.Flags().BoolP("previous", "p", false, "If true, print the logs for the previous instance of the container in a pod if it exists.")
 	cmd.Flags().StringVarP(&params.containerName, "container", "c", "", "Container name")
+	cmd.Flags().StringP("tail", "t", "all", "Number of lines to show from the end of the logs")
```

Comment:
Do not use a short flag here.
https://github.com/kubernetes/kubernetes/blob/master/docs/devel/kubectl-conventions.md#flag-conventions